message_buffer/programs/message_buffer/src/state/message_buffer.rs

```diff
@@ -227,23 +253,27 @@ mod test {
 
         let message_buffer = &mut MessageBuffer::new(0);
         let header_len = message_buffer.header_len as usize;
-        let message_buffer_bytes = bytes_of_mut(message_buffer);
-        // assuming account_info.data.len() == 10KB
-        let messages = &mut vec![0u8; 10_240 - header_len];
 
         let account_info_data = &mut vec![];
         let discriminator = &mut sighash("accounts", "MessageBuffer");
+        let destination = &mut vec![0u8; 10_240 - header_len];
+
+
```

Comment:
you've copy/pasted a lot of code to initialize message buffers in these tests. I recommend extracting out a helper method that sets everything up for you.

Reply:
actually these tests seem to have lots of repeated code blocks in them. I think this isn't ideal -- you should think about how to factor things into helper methods to reduce duplication.